star.wdl

```diff
@@ -75,26 +55,21 @@ input {
   String chimericjunctionSuffix = "Chimeric.out"
   String genereadSuffix = "ReadsPerGene.out"
   String? addParam
-  String modules = "star/2.7.6a hg38-star-index100/2.7.6a"
-  String chimOutType = "WithinBAM HardClip Junctions"
-  Int outFilterMultimapNmax = 50
-  Int chimScoreDropMax = 30
+  String modules = "star/2.7.3a hg38-star-index100/2.7.3a"
```

Comment:
This also looks suspicious. If we are going to use 2.7.3a and not 2.7.6a the README and CHANGELOG need to be changed

Reply:
I didn't update README, but I can do it. This PR is to migrate an old star workflow version 2.0.2 (not the latest one) to Vidarr.

Reply:
updated